R/queue.R

```diff
@@ -29,9 +29,11 @@ Queue <- R6::R6Class("Queue", #nolint
       self$controller <- rrq::rrq_controller(
         queue_id %||% orderly_queue_id()
       )
-      worker_config <- rrq::rrq_worker_config(heartbeat_period = 10)
+      log_dir_name <- "runner-logs"
+      dir.create(log_dir_name, showWarnings = FALSE)
+      worker_config <- rrq::rrq_worker_config(heartbeat_period = 10, logdir = log_dir_name)
```

Comment:
i dont think the logs should be at root level, this is the git repo, we've put all the worker stuff in `.packit` when the workers come up, i think `.packit/logs` is a suitable place because then we'll have `.packit/workers` for the worker roots and `.packit/logs` for the logs

but maybe lets wait for what rich thinks

Reply:
@richfitz so had another think about this, 2 potential options can be:
1. `./.packit/logs` as log dir
1. we can have the logs dir in `/worker-logs` at root level in the docker container which could be a logs volume shared by the workers? that could let us persist logs between restarts

Reply:
I really don't mind, so long as packit itself can't read them directly - only over the api.  You could make this another command line switch perhaps, though it really only needs setting before the first worker _ever_ turns up, it doesn't need doing each time as it is persisted in redis.

I wonder if using `.packit/` will be confusing as this is an orderly detail so instead do

```
/logs/worker
/logs/task
```

as your directory, and add `VOLUME /logs` to the dockerfile?